WordPress/Classes/ViewRelated/Me/All Domains/Views/AllDomainsListViewController.swift

```diff
@@ -139,6 +143,18 @@ final class AllDomainsListViewController: UIViewController {
     private func navigateToAddDomain() {
 
     }
+
+    private func navigateToDomainDetails(with domain: Domain) {
+        guard let url = domain.wpcomDetailsURL, let navigationController = navigationController else {
+            self.crashLogger.logMessage("Failed to navigate to Domain Details screen from All Domains screen", level: .error)
+            return
+        }
+        let destination = DomainDetailsWebViewController(
+            url: url,
+            analyticsSource: "all-domains"
+        )
+        navigationController.pushViewController(destination, animated: true)
```

Comment:
[Suggestion] This can make testing with a sandbox easier
```suggestion
        destination.configureSandboxStore {
            navigationController.pushViewController(destination, animated: true)
        }
```